spec/Error/NotAllCallsOccurredError_spec.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+describe('NotAllCallsOccurredError', () => {
+  let NotAllCallsOccurredError = require('../../src/Error/NotAllCallsOccurredError.js');
+
+  it('should stringify correctly', () => {
+    expect(new NotAllCallsOccurredError([], []).message).toEqual('Not all calls occurred');
```

Comment:
How about a quick check to make sure it stringifies the calls that did and didn't occur?